pkg/master/master.go

```diff
@@ -438,7 +439,9 @@ func (m *Master) InstallAPIs(apiResourceConfigSource serverstorage.APIResourceCo
 
 	if err := m.GenericAPIServer.InstallAPIGroups(apiGroupsInfo...); err != nil {
 		klog.Fatalf("Error in registering group versions: %v", err)
```

Comment:
`klog.Fatalf` exits the process. prefer `return fmt.Errorf("error registering group versions: %v", err)`